packages/react-data-grid/src/utils/__tests__/viewportUtils.spec.ts

```diff
@@ -46,22 +69,22 @@ describe('viewportUtils', () => {
   describe('getNonFrozenRenderedColumnCount', () => {
     const viewportWidth = 100;
 
-    const getCols = () => [
-      { key: 'col1', width: 20, left: 0 },
-      { key: 'col2', width: 20, left: 20 },
-      { key: 'col3', width: 20, left: 40 },
-      { key: 'col4', width: 20, left: 60 },
-      { key: 'col5', width: 20, left: 80 },
-      { key: 'col6', width: 1, left: 100 },
-      { key: 'col7', width: 1, left: 101 },
-      { key: 'col8', width: 1, left: 102 },
-      { key: 'col9', width: 1, left: 103 }
+    const getCols = (): Column[] => [
+      { key: 'col1', name: 'col1', width: 20, left: 0, onCellChange() {} },
```

Comment:
It seems like `onCellChange` is special prop for `Checkbox` column. It is not used anywhere else

https://github.com/adazzle/react-data-grid/blob/next/packages/react-data-grid/src/ReactDataGrid.js#L736
https://github.com/adazzle/react-data-grid/blob/next/packages/react-data-grid/src/common/editors/CheckboxEditor.tsx#L13

May be mark it as an optional prop? We need to revisit `Checkbox` formatter